lrauv_system_tests/worlds/tilted_tethys_no_damping.sdf

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" ?>
+<!--
+  Development of this module has been funded by the Monterey Bay Aquarium
+  Research Institute (MBARI) and the David and Lucile Packard Foundation
+-->
+<sdf version="1.6">
+  <world name="tilted_tethys_no_damping">
+    <gravity>0 0 -9.8</gravity>
+    <scene>
+      <!-- For turquoise ambient to match particle effect -->
+      <ambient>0.0 1.0 1.0</ambient>
+      <!-- For default gray ambient -->
+      <!--background>0.8 0.8 0.8</background-->
+      <background>0.0 0.7 0.8</background>
+    </scene>
+
+    <physics name="1ms" type="dart">
+      <max_step_size>0.02</max_step_size>
+      <real_time_factor>0</real_time_factor>
+    </physics>
+    <plugin
+      filename="ignition-gazebo-physics-system"
+      name="ignition::gazebo::systems::Physics">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-user-commands-system"
+      name="ignition::gazebo::systems::UserCommands">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-scene-broadcaster-system"
+      name="ignition::gazebo::systems::SceneBroadcaster">
+    </plugin>
+
+    <plugin
+      filename="ignition-gazebo-buoyancy-system"
+      name="ignition::gazebo::systems::Buoyancy">
+      <!-- TODO: fix behaviour near surface, see https://github.com/osrf/lrauv/issues/102
```

Comment:
We don't need this TODO anymore. This file in any case should be with uniform buoyancy since we only want to use this to test the righting moment on the vehicle.

Reply:
Removed in https://github.com/osrf/lrauv/pull/197/commits/4a749dcd1c2e3bf2405b136702d56b9cc6b9d3a6.